buildroot-external/board/hardkernel/odroid-m1s/meta

```diff
@@ -0,0 +1,12 @@
+BOARD_ID=odroid-m1s
+BOARD_NAME="Hardkernel ODROID-M1S"
+CHASSIS=embedded
+BOOTLOADER=uboot
+KERNEL_FILE=Image
+BOOT_SYS=gpt
+BOOT_SIZE=16M
+BOOT_SPL=true
+BOOT_SPL_SIZE=16M
+BOOT_ENV_SIZE=0x8000
+SUPERVISOR_MACHINE=odroid-m1
```

Comment:
I think Agners said we would share same supervisor etc, its mainly only because of the TPL DDR init code differing that mandated a new board.

Reply:
Granted, we typically create a separate Supervisor machine for each board. However, we debated just recently if that still make sense. 

In practice the things which requires board specific Supervisor machine is if we need board specific user space software. E.g. to leverage hardware accelerators or similar. Often these things are SoC specific.

We do have very little boards specific things in the Core container today, e.g. all Raspberry Pi board containers install the same packets in Core (see https://github.com/home-assistant/core/tree/dev/machine). But then, it only creates a small board specific layer, so it isn't a big burden for the build systems etc. And it would allow to create a board specific changes, if we really want to. So we decided to keep the system in place as is.

However, since ODROID-M1 and ODROID-M1S use the same SoC family with the same NPU and multi-media engine, and the board is from the same vendor, I'd say it is not worth making the distinction here. And if we really need to, we can introduce this in a later update.